queries/angular/highlights.scm

```diff
@@ -68,6 +68,9 @@
 ((control_keyword) @keyword.exception
   (#eq? @keyword.exception "error"))
 
+((control_keyword) @keyword
+  (#eq? @keyword "let"))
```

Comment:
Wouldn't it be easier to capture `(control_keyword) @keyword` first and then override the special cases?

Reply:
makes sense